silo/tests/test.py

```diff
@@ -817,3 +818,42 @@ def test_deleteAuto(self):
         self.assertFalse(silo)
         self.assertFalse(read)
         self.assertTrue(deleted_silos)
+
+
+class TestCleanKeys(TestCase):
```

Comment:
This test is called `TestCleanKeys` but it's doing like an integration test that looks weird to me. I would expect this to test only the function `cleanKey` instead: https://github.com/toladata/TolaTables/blob/0fc4e06510208639734fe7623e2581349d3a686f/tola/util.py#L183